docs/team/ian-from-dover.md

```diff
@@ -5,43 +5,53 @@ title: Ian Hong's Project Portfolio Page
 
 ### Project: PeopleSoft
 
-PeopleSoft lets HR managers of companies handle their people.
+PeopleSoft is a Payroll management app for companies offering contractor-based services.
```

Comment:
I think it would be more accurate to say handling as opposed to offering? Not sure, what do you think?